ITISFoundation/osparc-simcore
# qxapp -> osparc refactor

### Potentially breaking changes

qxapp -> osparc refactor

### How to test
```
make build-devel
make up-devel
```
and/or
```
make build
make up
```
and see that everything works fine
